### DISCUSSION TITLE
Deauthorized Xero Connection

### DISCUSSION THREAD

**Discussion by FreddieRidell** [codatio/codat-docs#256] [Q&A] [answered]
Hi, I have a company with a Xero connection which is Deauthorized, what could have caused this?

**Comment by pzaichkina** [comment, marked as answer]
Hi Freddie, 

When we attempt to refresh the user token for a Company, sometimes we receive a response from Xero of `unauthorized`. 

Unfortunately, Xero doesn't provide details specifying _why_ access became unauthorized. It could mean that there was a change in the permissions of the user who initially connected the account, or that the access had been revoked by the customer. In order for the customer to continue syncing, they will need to relink to Xero.

Hope this helps.

Polina,
Tech writer, Codat

**Comment by FreddieRidell** [comment]
Thanks for the quick reply Polina!
 I'll get in contact with the customer and tell them they'll need to re-link to Xero
